<?xml version="1.0" ?>
<testsuite errors="1" failures="1" name="test.testcases.MixedTestCase" tests="3" time="0.000">
	<testcase classname="test.testcases.MixedTestCase" name="test_success" time="0.000"/>
	<testcase classname="test.testcases.MixedTestCase" name="test_failure" time="0.000">
		<failure message="" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "{{PATH}}/test/testcases.py", line 34, in test_failure
    self.assertTrue(None)
AssertionError
]]>		</failure>
	</testcase>
	<testcase classname="test.testcases.MixedTestCase" name="test_errord" time="0.000">
		<error message="unsupported operand type(s) for +: 'NoneType' and 'int'" type="TypeError">
<![CDATA[Traceback (most recent call last):
  File "{{PATH}}/test/testcases.py", line 37, in test_errord
    None + 1
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[some text
]]>	</system-out>
	<system-err>
<![CDATA[another text
]]>	</system-err>
</testsuite>
